src/client/routes/Vaults/index.tsx

```diff
@@ -40,31 +40,23 @@ import {
 import { getConfig } from '@config';
 import { VaultView } from '@src/core/types';
 
-const SearchBarContainer = styled.div`
-  margin: 1.2rem;
-`;
-
-const Row = styled.div`
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: flex-start;
-  grid-gap: ${({ theme }) => theme.layoutPadding};
-  flex-wrap: wrap;
-  width: 100%;
-`;
+// const Row = styled.div`
+//   display: flex;
+//   flex-wrap: wrap;
+//   justify-content: flex-start;
+//   grid-gap: ${({ theme }) => theme.layoutPadding};
+//   flex-wrap: wrap;
+//   width: 100%;
+// `;
 
```

Comment:
Consider removing

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6